index.md

```diff
@@ -1034,7 +1032,6 @@ identification purposes only and does not constitute endorsement.)</small>
 1. Milan Pässler (Chaos Computer Club, NixOS contributor)
 1. Milo Casagrande
 1. Ming-ting Yao Wei (Debian Developer, former LXTerminal developer)
-1. Minho Ryang (@minhoryang on Github)
```

Comment:
@esklarski hmm ...

Reply:
Whoa... What... I appologise and have no idea where those deletions came from. Removing this request.